torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -2766,6 +2856,12 @@ def _post_backward_hook(self, param: Parameter, *unused: Any) -> None:
             # then subsequent hook callbacks will see POST state.
             self._assert_state([TrainingState_.BACKWARD_PRE, TrainingState_.BACKWARD_POST])
             self.training_state = TrainingState_.BACKWARD_POST
+
+            if (self._use_param_exec_order_policy() and self._param_exec_order_prep_stage):
```

Comment:
nit: Unlike C/C++, we should not use parentheses around conjunctions. I got called out on this last summer 😆